src/svd/device.py

```diff
@@ -1395,37 +1564,41 @@ def _extract_register_info(
     else:
         address_bounds = base_address_bounds
 
+    if base_reg_props.reset_value is None:
+        raise ValueError("TODO")
```

Comment:
Should we link this to an issue so it doesn't go overlooked and forgotten?

Reply:
This was another case of forgetting to update something. I added better handling for this now.